tests/unit/api/test_delete_importable_dataset_api.py

```diff
@@ -0,0 +1,16 @@
+from flask import url_for
+import os
+
+def test_delete_importable_datasets_api(flask_app):
+    f = open("tests/resources/input_directory/DATASET_THAT_SHOULD_BE_DELETED.tar", "x")
+    response = flask_app.delete(
+        url_for("importable_datasets_api.delete_importable_datasets", dataset_name="DATASET_THAT_SHOULD_BE_DELETED.tar")
```

Comment:
dataset_name should be sent to the api without the .tar extension. the app can add this in the remove operation 👍🏻